nebari/provider/cicd/gitlab.py

```diff
@@ -43,20 +43,20 @@ def gen_gitlab_ci(config):
     commit_render = config["ci_cd"].get("commit_render", True)
     before_script = config["ci_cd"].get("before_script")
     after_script = config["ci_cd"].get("after_script")
-    pip_install = pip_install_qhub(config["qhub_version"])
+    pip_install = pip_install_nebari(config["nebari_version"])
 
     render_vars = {
-        "COMMIT_MSG": "qhub-config.yaml automated commit: {{ '$CI_COMMIT_SHA' }}",
+        "COMMIT_MSG": "nebari-config.yaml automated commit: {{ '$CI_COMMIT_SHA' }}",
     }
 
     script = [
         f"git checkout {branch}",
         f"{pip_install}",
-        "qhub deploy --config qhub-config.yaml --disable-prompt --skip-remote-state-provision",
+        "nebari deploy --config nebari-config.yaml --disable-prompt --skip-remote-state-provision",
     ]
 
     commit_render_script = [
-        "git config user.email 'qhub@quansight.com'",
+        "git config user.email 'nebari@quansight.com'",
```

Comment:
are we still using this email? need to make sure this is a valid email and preferably set from within environment variables

Reply:
Can we open an issue for this? I'm afraid that including too many additional changes in one PR...